tests/stats_test.py

```diff
@@ -1,4 +1,4 @@
-import numpy as np
+ import numpy as np
```

Comment:
You added a space here, probably unintentionally
```suggestion
import numpy as np
```

Reply:
Updated